sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/serializer/DeserializationHelpers.java

```diff
@@ -9,7 +12,7 @@
 
 import static com.azure.core.util.serializer.TypeReference.createInstance;
 
-public class DeserializationHelpers {
+public final class DeserializationHelpers {
```

Comment:
xml comments? :)

Reply:
Good idea. But this is Java, so I'll do javadocs ;)

Reply:
yes that :)